orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/StartStageHandler.kt

```diff
@@ -76,6 +76,7 @@ open class StartStageHandler
             val exceptionDetails = exceptionHandlers.shouldRetry(e, stage.getName())
             if (exceptionDetails?.shouldRetry ?: false) {
               log.warn("Error planning ${stage.getType()} stage for ${message.executionType.simpleName}[${message.executionId}]")
+              message.setAttribute(MaxAttemptsAttribute(5))
```

Comment:
Actual problem I wanted to solve ... an upper bounds on the # of times this message will be retried.

`queue.push()` ends up assigning a new `UUID` so for all intents and purposes it's a brand new message. Suspect this is the case as the original message is in the process of being `ack()'d`?